app/controllers/host_reports_controller.rb

```diff
@@ -2,4 +2,44 @@
 
 class HostReportsController < ::ApplicationController
   include Foreman::Controller::AutoCompleteSearch
+
+  def show
+    @host_report = resource_scope.find(params[:id])
+
+    return not_found unless @host_report
+
+    respond_to do |format|
+      format.html do
+        render '/react/index'
+      end
+      format.json do
+        render json: {
+          host_report: {
+            id: @host_report.id,
+            body: JSON.parse(@host_report.body),
+            host: {
+              id: @host_report.host.id,
+              name: @host_report.host.name,
+            },
+            proxy: {
+              id: @host_report.proxy&.id,
+              name: @host_report.proxy&.name,
+            },
+            reported_at: @host_report.reported_at.utc.to_s,
```

Comment:
Is it okay to hardcode this to UTC?

Reply:
I've changed that to be the same value we store. Then it will be presented by javascript based on browser settings (lang/timezone).